### DISCUSSION TITLE
Meteorological Data Units

### DISCUSSION THREAD

**Discussion by bulutenesemre** [meteostat/meteostat-python#60] [Q&A]
Hey Good Dev,

I can see https://dev.meteostat.net/formats.html#meteorological-parameters here,  Sunshine duration : Minutes/Hours is not clear from my side and it mostly has value 0 . Can we make it clearify ?

**Comment by clampr** [comment]
I updated the docs. It's minutes on all endpoints now.

**Comment by bulutenesemre** [comment]
So what does it mean exactly? How can I use it? For example if that value is smaller than 30 can we say its dark?

**Reply by clampr** [reply]
Not really. It's not dark if the sun is covered by clouds. It just means that the sun isn't "shining". From Wikipedia:

>  It is a general indicator of cloudiness of a location, and thus differs from insolation, which measures the total energy delivered by sunlight over a given period.

**Reply by bulutenesemre** [reply]
That was just a sample. So do you have any advice how to classify that value of Sunshine duration?
